src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.AppHost/xxENSONOxx.xxSTACKSxx.AppHost.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsAspireHost>true</IsAspireHost>
+    <UserSecretsId>a9ea3488-8409-4541-bfa2-f65cc3ba6449</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Aspire.Hosting.AppHost" Version="8.1.0" />
```

Comment:
I had to change this to `8.2.0` in order for the build to work with the `dotnet workload install aspire`. I imagine there is a way install a specific version though